nutkit/protocol/responses.py

```diff
@@ -23,7 +23,9 @@
 
 class RunTest:
     """ Response to StartTest indicating that the test can be started"""
-    pass
+
+    def __init__(self, capabilities=[]):
```

Comment:
This is a bad idea because lists are mutable and that default argument will be the same list for every call.

```python
def __init__(self, capabilities=None):
    if capabilities is None:
        capabilities = []
    ...
```

Reply:
Thanks, addressed.